ProjectLighthouse.Servers.GameServer/Controllers/Slots/SlotsController.cs

```diff
@@ -157,7 +157,9 @@ public async Task<IActionResult> NewestSlots([FromQuery] int pageStart, [FromQue
 
         GameVersion gameVersion = token.GameVersion;
 
+        DateTimeOffset cutoff = DateTimeOffset.Now.AddDays(-31);
         List<SlotBase> slots = (await this.database.Slots.ByGameVersion(gameVersion, false, true)
+            .Where(s => s.FirstUploaded >= cutoff.ToUnixTimeMilliseconds())
```

Comment:
Why are you adding a cutoff to the newest slots? They are already going to be sorted in chronological order so you will see the newest levels first.